renderer/mkDocsAdmonition.go

```diff
@@ -114,8 +115,11 @@ func (r *ConfluenceMkDocsAdmonitionRenderer) renderMkDocsAdmonition(writer util.
 		if _, err := writer.Write([]byte(prefix)); err != nil {
 			return ast.WalkStop, err
 		}
-		if string(n.Title) != "" {
-			titleHTML := fmt.Sprintf("<p><strong>%s</strong></p>\n", string(n.Title))
+
+		title := strings.Trim(string(n.Title), "\"")
```

Comment:
This will likely trim too many quotes. We only want to remove the first and last one, I guess?